src/features/navigation/RootNavigator/routes.ts

```diff
@@ -56,6 +57,10 @@ const routes: Array<Route> = [
     name: 'EighteenBirthday',
     component: EighteenBirthday,
   },
+  {
+    name: 'FavoritesFilters',
+    component: Filters,
```

Comment:
Je pense qu'il faut expliciter le naming du composant car sinon c'est ambigu
```suggestion
    component: FavoriteFilters,
```

Reply:
OK